GraficaEngine/GraficaEngine/Core/Scene.h

```diff
@@ -13,13 +13,17 @@
 #include "../Renderer/Skybox.h"
 #include "../Physics/Collider.h"
 
+#include "../Renderer/MeshRenderer.h"
+#include "../Renderer/ShadowRenderer.h"
+#include "../Renderer/TextRenderer.h"
+
 namespace Engine
 {
 	class Scene
 	{
 	private:
 		Camera *_activeCamera;
-		std::map<std::string, Camera *> _cameras;
+		std::unordered_map<std::string, Camera *> _cameras;
```

Comment:
I'm intrigued, why is it needed that this is an unordered map? The map is ordered by default?

Reply:
The map is ordered by default. Causing the insertions to be done in order n